mypy/test/data.py

```diff
@@ -647,8 +662,12 @@ class DataSuite:
     optional_out = False  # type: bool
     native_sep = False  # type: bool
 
-    def __init__(self, *, update_data: bool) -> None:
-        self.update_data = update_data
+    # Assigned from MypyDataCase.runtest
+    update_data = False  # type: bool
```

Comment:
Style nit: redundant type annotation (you can also remove the `str` and `bool` annotations above).